### PR TITLE
docs: v4 ended

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |       -    |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — DISMISSED** [review]

**Review by kiner-tang — DISMISSED** [review]

File: docs/blog/v4-ood.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: 📢 v4 维护周期截止
+date: 2024-02-27
+author: zombieJ
+---
+
+根据我们的发布时间表，Ant Design v4 版本的维护期已正式结束。这意味着，除非发现了重大的缺陷或其他严重问题，我们不会再推出任何新的更新或补丁版本给 v4。我们强烈建议用户尽快升级到Ant Design v5 版本。
```
Comment (afc163) [inline docs/blog/v4-ood.zh-CN.md:7]:
```suggestion
根据我们的发布时间表，Ant Design v4 版本的维护期已正式结束。这意味着，除非发现了重大的缺陷或其他严重问题，我们不会再推出任何新的更新或补丁版本给 v4。我们强烈建议用户尽快升级到 Ant Design v5 版本。
```

File: docs/blog/v4-ood.en-US.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+You will get a series of benefits from upgrading to v5, including more powerful design system theme capabilities, richer component functionality, performance optimization, and more. In addition, v5 also includes many issue fixes and user experience improvements based on community feedback. We will continue to provide continuous support and updates on v5 to ensure that developers can get the best user experience.
+
+For a smooth transition to v5, we provide detailed upgrade guides and documentation. We understand that the upgrade process may involve a lot of work, so our team is also working hard to make this process as seamless and simple as possible. If you encounter any problems during the upgrade process, please feel free to submit an issue on GitHub.
+
+Please visit our official website or directly view the v5 upgrade documentation to start your upgrade journey and enjoy the latest Ant Design experience.
```
Comment (rianonehub) [inline docs/blog/v4-ood.en-US.md:13]: Please visit our official website or view the v5 upgrade documentation directly to start your upgrade journey and enjoy the latest Ant Design experience.

**Review by rianonehub — CHANGES_REQUESTED** [review]

File: docs/blog/v4-ood.en-US.md (hunk 2 above)
Comment (yoyo837) [reply docs/blog/v4-ood.en-US.md:13]: > Please visit our official website or view the v5 upgrade documentation directly to start your upgrade journey and enjoy the latest Ant Design experience.

Welcome to make suggestions directly.

Comment (rianonehub) [reply docs/blog/v4-ood.en-US.md:13]: sorry my bad 😊

File: docs/blog/v4-ood.zh-CN.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: 📢 v4 维护周期截止
+date: 2024-02-27
+author: zombieJ
+---
+
+根据我们的发布时间表，Ant Design v4 版本的维护期已于 2023 年年底正式结束。这意味着，除非发现了重大的缺陷或其他严重问题，我们不会再推出任何新的更新或补丁版本给 v4。我们强烈建议用户尽快升级到Ant Design v5 版本。
+
+升级至 v5 将为您带来一系列的好处：包括对更强大的设计系统主题能力、更为丰富的组件功能、性能优化等等。此外，v5还包含了许多社区反馈的问题修复和用户体验的改进。我们将继续在 v5 上提供持续的支持和更新，确保开发者可以获得最佳的使用体验。
```
Comment (kiner-tang) [inline docs/blog/v4-ood.zh-CN.md:9]:
```suggestion
升级至 v5 将为您带来一系列的好处：包括对更强大的设计系统主题能力、更为丰富的组件功能、性能优化等等。此外，v5 还包含了许多社区反馈的问题修复和用户体验的改进。我们将继续在 v5 上提供持续的支持和更新，确保开发者可以获得最佳的使用体验。
```

File: docs/blog/v4-ood.en-US.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: 📢 v4 surpassed maintenance period
+date: 2024-02-27
```
Comment (li-jia-nan) [inline docs/blog/v4-ood.en-US.md:3]:
```suggestion
date: 2024-02-28
```

File: docs/blog/v4-ood.zh-CN.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: 📢 v4 维护周期截止
+date: 2024-02-27
```
Comment (li-jia-nan) [inline docs/blog/v4-ood.zh-CN.md:3]:
```suggestion
date: 2024-02-28
```

File: docs/blog/v4-ood.zh-CN.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+title: 📢 v4 维护周期截止
+date: 2024-02-28
+author: zombieJ
+---
+
+根据我们的发布时间表，Ant Design v4 版本的维护期已于 2023 年年底正式结束。这意味着，除非发现了重大的缺陷或其他严重问题，我们不会再推出任何新的更新或补丁版本给 v4。我们强烈建议用户尽快升级到 Ant Design v5 版本。
```
Comment (kiner-tang) [inline docs/blog/v4-ood.zh-CN.md:7]:
```suggestion
根据我们的发布时间表，Ant Design v4 版本的维护期已于 2023 年年底正式结束。这意味着，除非发现了重大的缺陷或其他严重问题，我们不会再为 v4 推出任何新的更新或补丁版本。我们强烈建议用户尽快升级到 Ant Design v5 版本。
```

**Review by li-jia-nan — DISMISSED** [review]

**Review by li-jia-nan — DISMISSED** [review]

**Review by kiner-tang — DISMISSED** [review]

**Review by crazyair — DISMISSED** [review]

File: docs/blog/v4-ood.zh-CN.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+title: 📢 v4 维护周期截止
+date: 2024-02-28
+author: zombieJ
+---
+
+根据我们的发布时间表，Ant Design v4 版本的维护期已于 2023 年年底正式结束。这意味着，除非发现了重大的缺陷或其他严重问题，我们不会再为 v4 推出任何新的更新或补丁版本。我们强烈建议用户尽快升级到 Ant Design v5 版本。
+
+升级至 v5 将为您带来一系列的好处：包括对更强大的设计系统主题能力、更为丰富的组件功能、性能优化等等。此外，v5 还包含了许多社区反馈的问题修复和用户体验的改进。我们将继续在 v5 上提供持续的支持和更新，确保开发者可以获得最佳的使用体验。
+
+为了帮助您平滑迁移到 v5，我们提供了详尽的升级指南和文档。我们了解升级过程可能会涉及到不少工作量，因此我们的团队也在努力使这个过程尽可能无缝和简便。如果你在升级过程中遇到任何问题，欢迎在 github 中提交 issue 反馈。
```
Comment (li-jia-nan) [inline docs/blog/v4-ood.zh-CN.md:11]:
```suggestion
为了帮助您平滑迁移到 v5，我们提供了详尽的升级指南和文档。我们了解升级过程可能会涉及到不少工作量，因此我们的团队也在努力使这个过程尽可能无缝和简便。如果你在升级过程中遇到任何问题，欢迎在 GitHub 中提交 issue 反馈。
```

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**